src/lib/helpers.ts

```diff
@@ -29,6 +29,13 @@ export const getEventPath = (url: string) => {
   return (`/${eventPath}`)
 }
 
+export const getNewsPath = (url: string) => {
```

Comment:
You could combine `getEventPath` and this function and rename the function to `getPath` because the functions have the same code.